draft-ietf-rats-corim.md

```diff
@@ -511,7 +513,8 @@ The CoRIM MUST be signed by the CoRIM creator.
 
 The following CDDL specification defines a restrictive subset of COSE header
 parameters that MUST be used in the protected header alongside additional
-information about the CoRIM encoded in a `corim-meta-map` ({{sec-corim-meta}}).
+information about the CoRIM encoded in a `CWT-Claims` ({{-CWT_CLAIMS_COSE}})
+or as a legacy alternative in a `corim-meta-map` ({{sec-corim-meta}}).
```

Comment:
```suggestion
information about the CoRIM encoded in a `corim-meta-map` ({{sec-corim-meta}}) or alternatively in a `CWT-Claims` ({{-CWT_CLAIMS_COSE}}).
```
The existing wording seems to suggest the current sec-corim-meta is deprecated or might be deprecated in the future. We didn't discuss this as tobe deprecated, so different wording is needed.

Reply:
We did discuss this at length, there are traces in writing as well: https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/479#discussion_r2288360365